<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMFittingWidget</class>
 <widget class="QFrame" name="KMFittingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#KMFittingWidget{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageFitting">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidgetFitting">
         <widget class="QWidget" name="pageMain">
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="frameTable_m">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frameBtnBox_m">
             <property name="minimumSize">
              <size>
               <width>256</width>
               <height>64</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame#frameBtnBox_m {
	background-color: rgb(160, 160, 160);
}

QToolButton {
	color:white;
	font: 9pt &quot;微软雅黑&quot;;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="toolButtonAdd">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>91</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/kitmethod_icons/plus.png</normaloff>:/kitmethod_icons/plus.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextUnderIcon</enum>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonRemove">
                <property name="minimumSize">
                 <size>
                  <width>91</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Del</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/kitmethod_icons/remove.png</normaloff>:/kitmethod_icons/remove.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextUnderIcon</enum>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonFit">
                <property name="minimumSize">
                 <size>
                  <width>91</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Fit</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/kitmethod_icons/fit.png</normaloff>:/kitmethod_icons/fit.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextUnderIcon</enum>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonSwitch">
                <property name="minimumSize">
                 <size>
                  <width>91</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Switch</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/kitmethod_icons/change.png</normaloff>:/kitmethod_icons/change.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextUnderIcon</enum>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageAdd">
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="frameHeader">
             <property name="styleSheet">
              <string notr="true">QFrame#frameHeader{
	background-color: rgb(160, 160, 160);
}

QLabel{
	font: 12pt &quot;微软雅黑&quot;;
	color:white;
	background: transparent;
}

QSpinBox, QDoubleSpinBox, QDateTimeEdit {
	border: 1px solid white;
	font: 12pt &quot;微软雅黑&quot;;
	color:white;
	background: transparent;
}

QSpinBox::up-button, QDoubleSpinBox::up-button, QDateTimeEdit::up-button {
	subcontrol-origin:border;
	subcontrol-position:right;
	padding-left: 10px;
	image: url(:/protein_icons/arrow_right.png);
	width: 24px;height: 28px; right:5px;
}
QSpinBox::down-button, QDoubleSpinBox::down-button, QDateTimeEdit::down-button {
	subcontrol-origin:border;
	subcontrol-position:left;
	padding-right: 10px;
	image: url(:/protein_icons/arrow_left.png);
	width: 24px;height: 28px; left:5px;
}
QSpinBox::up-button:pressed, QDoubleSpinBox::up-button:pressed, QDateTimeEdit::up-button:pressed {
	subcontrol-origin:border;
	subcontrol-position:right;
	image: url(:/protein_icons/arrow_right.png);
	width: 23px;height: 27px;
}
QSpinBox::down-button:pressed, QDoubleSpinBox::down-button:pressed, QDateTimeEdit::down-button:pressed {
	subcontrol-position:left;
	image: url(:/protein_icons/arrow_left.png);
	width: 23px;height: 27px;
}
QSpinBox::disabled, QDoubleSpinBox::disabled, QDateTimeEdit::disabled {
	color:darkgray;
}
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <item>
               <widget class="QLabel" name="labelConcentration">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="text">
                 <string>Sample Concentration:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBoxConcentration">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="suffix">
                 <string> mg/ml</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frameKeyboard">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frameBtnBox_a">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>55</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame#frameBtnBox_a{
	background-color: rgb(160,160,160);
}

QToolButton {
	color:white;
	font: 18pt &quot;微软雅黑&quot;;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="toolButtonCancel_a">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Cancel</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonOk_a">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Ok</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageRemove">
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="radioButtonCheckAll">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>48</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton{
	color: Blue;
	font: 18pt &quot;微软雅黑&quot;;
	background-color: rgb(160, 160, 160);
}</string>
             </property>
             <property name="text">
              <string>Check All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frameTable_r">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frameBtnBox_r">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>55</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame#frameBtnBox_r{
	background-color: rgb(160,160,160);
}

QToolButton {
	color:white;
	font: 18pt &quot;微软雅黑&quot;;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="toolButtonCancel_r">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Cancel</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonOk_r">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Ok</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageMeasure">
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frameTableMeasure">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameBtnBoxMeasure">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame#frameBtnBoxMeasure{
	background-color: rgb(160,160,160);
}

QToolButton {
	color:white;
	font: 18pt &quot;微软雅黑&quot;;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButtonBack">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonClear">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="kitmethodwindow.qrc"/>
 </resources>
 <connections/>
</ui>
